pkg/siteclone_watcher.go

```diff
@@ -46,7 +46,7 @@ func (w scWatcher) OnUpdate(oldObj, newObj interface{}) {
 	}
 	nr, nm := n.Ready()
 	or, om := o.Ready()
-	if nr == or && nm == om {
+	if nr == or && nm == om && n.Error() == o.Error() {
```

Comment:
and this was the culprit. The `old` status of `SiteClone` was also not ready as is the `new` status, but the error message differs.